src/GlobalStyle.js

```diff
@@ -8,12 +8,17 @@ const GlobalStyle = createGlobalStyle`
     font-family: Montserrat, sans-serif;
     font-weight: 700;
     font-size: 16px;
-    height: 100%;
+    overflow-y: scroll;
```

Comment:
This shouldn't be required. What is the reason you put `overflow-y: scroll;` here?

Reply:
overflow-y: scroll avoids the horizontal jump that you have when you have pages that overflows the screen and others that not. It's something that maybe here is not so important, but in layouts where you don't want weird transitions.

Reply:
Ok cool, then leave it in

Reply:
:) actually, as it is now, all pages are higher than 100%, so it can be something to work in later.